iis-wf/iis-wf-primary/src/main/resources/eu/dnetlib/iis/wf/primary/main/oozie_app/workflow.xml

```diff
@@ -316,6 +321,11 @@
             <value>$UNDEFINED$</value>
             <description>document to organization trust level threshold</description>
         </property>
+        <property>
+            <name>export_trust_level_threshold_citation_relations</name>
```

Comment:
Using `export_action_set_id_document_referencedDocuments` was my first pick but I've changed it because I thought it will interfere with current export. I think it's better to change the name now because the task is already on the table. So I'm changing to citation_relations` to `document_referencedDocuments`.